db/config.cc

```diff
@@ -530,6 +530,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     /* Note: does not exist on the listing page other than in above comment, wtf? */
     , commitlog_sync_batch_window_in_ms(this, "commitlog_sync_batch_window_in_ms", value_status::Used, 10000,
         "Controls how long the system waits for other writes before performing a sync in \"batch\" mode.")
+    , commitlog_max_data_lifetime_in_s(this, "commitlog_max_data_lifetime_in_s", value_status::Used, 0,
+        "Controls how long data remains in commit log before the system tries to evict it to sstable, regardless of usage pressure. (0 disables)")
```

Comment:
I think that nobody will enable it until they noticed something bad like tombstone accumulation.
So I suggest we have some non-zero default like 3600 seconds.
As for the name, I wouldn't abbreviate it, and use `commitlog_max_data_lifetime_in_seconds`, which is clearer if you're not a developer.

Reply:
I set it to 24h as per @avikivity :s req.